lib/levelup.js

```diff
@@ -116,7 +112,12 @@ LevelUP.prototype.open = function (callback) {
   this._status = 'opening'
   this.db      = new DeferredLevelDOWN(this.location)
   dbFactory    = this.options.db || getLevelDOWN()
```

Comment:
Hmm, we could wrap this and throw a custom error, e.g.

```js
try {
  dbFactory = this.options.db || getLevelDOWN()
} catch (e) {
  throw new Error('missing db factory, you need to set options.db')
}
```

And add a test for it as well.